### PR TITLE
New Presentation Definition Guide

### PR DESCRIPTION
**Pull request by angiejones**
## What type of PR is this? (check all applicable)

- [x] 🧑‍💻 Code Snippet
- [x] 📖 Content
- [x] 🧪 Tests

## Description

This PR adds a new guide that covers how to create and validate Presentation Definitions

## Direct Link to Deploy Preview 

https://deploy-preview-1128--tbd-website-developer.netlify.app/docs/web5/build/verifiable-credentials/presentation-definition

## Added code snippets?
- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## What gif best describes this PR or how it makes you feel?

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206190035352034

### CODE REVIEW

File: site/docs/web5/build/verifiable-credentials/presentation-definition.mdx [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+```
+
+</details>
+
+
+### Properties
+
+* **id**: A required unique identifier that distinguishes this PD from others
+* **name**: An optional, human-friendly name for easier identification of the PD
+* **purpose**:  An optional description outlining why the information requested by the PD is needed
```
Comment (acekyd) [inline site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:126]: While listing these properties, group is missing. Although it's mentioned in the breakdown below.

Comment (angiejones) [reply site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:126]: these of the top-level properties. `group` is a property of `input_descriptors`

Comment (angiejones) [reply site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:126]: I can specify that these are top level properties in the doc, but do you think I should just add descriptions here for all the properties? That would include group, rule, count, min, max, filter, etc etc

Comment (angiejones) [reply site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:126]: i think i should make these clickable where they lead to the section that goes into more detail about it, and then from there, the reader can see the sub-properties for that property. what ya think?

Comment (acekyd) [reply site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:126]: I think mentioning top level ones should work and then go into more details later like you described. I like that.

Comment (angiejones) [reply site/docs/web5/build/verifiable-credentials/presentation-definition.mdx]: added! thanks Ace 😊

File: site/docs/web5/build/verifiable-credentials/presentation-definition.mdx [hunk 2]
```diff
@@ -0,0 +1,697 @@
...
+
+* [id](#id): A required unique identifier that distinguishes this PD from others
+* [name](#name-and-purpose): An optional, human-friendly name for easier identification of the PD
+* [purpose](#name-and-purpose):  An optional description outlining why the information requested by the PD is needed
+* [format](https://identity.foundation/presentation-exchange/spec/v1.0.0/#claim-format-designations):  The optional technical format requirements for the data to be submitted. *Example: { "jwt_vc": {"alg": ["ES256K", "ES384"]} }*
+* [input_descriptors](#input_selectors):  This is the most important property of the PD. It details the exact data requirements
+* [submission_requirements](#submission_requirements):  An optional feature that specifies which combinations of input descriptors are necessary to meet the requirements
+
+:::info
+For more in-depth information on the schema and options, refer to the [Presentation Defintion spec](https://identity.foundation/presentation-exchange/spec/v1.0.0/#json-schema-2).
```
Comment (angiejones) [inline site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:134]:
```suggestion
For more in-depth information on the schema and options, refer to the [Presentation Definition spec](https://identity.foundation/presentation-exchange/spec/v1.0.0/#json-schema-2).
```

File: site/docs/web5/build/verifiable-credentials/presentation-definition.mdx [hunk 3]
```diff
@@ -0,0 +1,697 @@
...
+          }
+        ]
+      }
+    },    
+  ]
+}
+```
+
+:::info
+The `filter` property offers a lot of flexibility. Refer to the [Presentation Defintion spec](https://identity.foundation/presentation-exchange/spec/v1.0.0/#json-schema-2) for more examples.
```
Comment (angiejones) [inline site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:334]:
```suggestion
The `filter` property offers a lot of flexibility. Refer to the [Presentation Definition spec](https://identity.foundation/presentation-exchange/spec/v1.0.0/#json-schema-2) for more examples.
```

File: site/docs/web5/build/verifiable-credentials/presentation-definition.mdx [hunk 4]
```diff
@@ -0,0 +1,697 @@
...
+    {
+      "name": "Employment and Academic/Certification Requirement",
+      "purpose": "Verify the applicant's employment history, and at least one of academic qualification or professional certification",
+    }
+  ]
+```
+
+#### rule
+
+The `rule` is a required property of `submission_requirements` and must be "all" or "pick" to indicate if all input descriptors in the Presentation must be satisfied, or if the applicant is allowed to pick a subset to satisfy.
```
Comment (angiejones) [inline site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:452]:
```suggestion
The `rule` is a required property of `submission_requirements` and must be "all" or "pick" to indicate if all input descriptors in the Presentation Definition be satisfied, or if the applicant is allowed to pick a subset to satisfy. 
```

File: site/docs/web5/build/verifiable-credentials/presentation-definition.mdx [hunk 5]
```diff
@@ -0,0 +1,697 @@
...
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1"
+    ],
+    "type": [
+      "VerifiableCredential",
+      "CertificationCredential"
+    ],
+    "id": "urn:uuid:7fba00eb-2f47-4113-aea6-4c5b5799f95f",
+    "issuer": "did:example:123456789abcdefghi",
+    "issuanceDate": "2023-01-05T16:06:00Z",
```
Comment (angiejones) [inline site/docs/web5/build/verifiable-credentials/presentation-definition.mdx:681]:
```suggestion
    "issuanceDate": "2023-05-01T16:06:00Z",
```

**Pull request merged**